pod/perldiag.pod

```diff
@@ -4502,9 +4502,15 @@ C<sysread()>ing a file, or when seeking past the end of a scalar opened
 for I/O (in anticipation of future reads and to imitate the behavior
 with real files).
 
+=item Old package separator "'" deprecated
+
+(W deprecated, syntax) You used the old package separator "'" in a
+variable, subroutine or package name.  Support for the old package
+separator will be removed in Perl 5.38.
```

Comment:
Shouldn't this say '5.40' as it's deprecated in 5.38 but removed according to the deprecation policy?

Reply:
Thanks, fixed.